src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java

```diff
@@ -87,6 +88,7 @@ private void setCaseDetails(CollectionCase collectionCase, Case caseDetails) {
     caseDetails.setLongitude(collectionCase.getAddress().getLongitude());
     caseDetails.setUprn(collectionCase.getAddress().getUprn());
     caseDetails.setRegion(collectionCase.getAddress().getRegion());
+    caseDetails.setCaseType("HH");
```

Comment:
I don't think this should be hard-coded should it?

Reply:
Case Type now taken from Case Created Event.